plugin.rb

```diff
@@ -27,22 +30,155 @@ class Engine < ::Rails::Engine
       engine_name PLUGIN_NAME
       isolate_namespace Salesforce
     end
+
+    %i{
+      leads
+      contacts
+    }.each do |type|
+      define_singleton_method("#{type}_group") do
+        Group.joins(:_custom_fields).find_by(_custom_fields: {name: group_custom_field(type), value: "t"})
```

Comment:
Instead of having to joins against the custom fields and using the name to find the group, perhaps we can use a hidden site setting to store the ID of the group while we seed the database?